Doc/library/re.rst

```diff
@@ -1110,6 +1110,10 @@ Exceptions
    .. versionchanged:: 3.5
       Added additional attributes.
 
+   .. versionchanged:: 3.12
```

Comment:
[3.12 is at feature freeze](https://discuss.python.org/t/python-3-12-beta-1-and-feature-freeze-is-here/26982?u=hugovk), so let's target 3.13:
```suggestion
   .. versionchanged:: 3.13
```